tests/integration/test_kinesis.py

```diff
@@ -53,10 +53,6 @@ def assert_consumers(count):
             ConsumerName=consumer_name)['ConsumerDescription']
         self.assertEqual(consumer_description_by_arn, consumer_description_by_name)
 
-        # delete non-existing consumer and assert 1 consumer
-        client.deregister_stream_consumer(StreamARN=stream_arn, ConsumerName='_invalid_')
```

Comment:
This test would fail in kinesis-mock with a ResourceNotFoundException because the consumer does not exist. We can remove this test.